financialaid/migrations/0003_added_timestamps.py

```diff
@@ -4,7 +4,7 @@
 
 import datetime
 from django.db import migrations, models
-
+import pytz
```

Comment:
Yes, I understand, but what about adding an extra migration instead of modifying this one?